src/service/auth.service.ts

```diff
@@ -29,7 +29,12 @@ class AuthService {
     // Generate email content (HTML format recommended for better formatting)
     const emailContent = resetPasswordTemplate(resetUrl);
 
-    await sendEmail(user.email, 'Password Reset', emailContent); // Send email using sendEmail function
+    await sendEmail({
+      to: user.email,
+      from: 'your@email.com',
```

Comment:
Why are we hardcoding the email here? Can't we pass the email argument from this method?

Reply:
> Why are we hardcoding the email here? Can't we pass the email argument from this method?

@Sagastein and @andremugabo  was the one worked on email templates, it shows that i've changed it because of the conflicts about models path i had in the file,,, i discussed it with @andremugabo he will add it in new fix PR he created.